include/oxen/quic/gnutls_crypto.hpp

```diff
@@ -303,6 +407,7 @@ namespace oxen::quic
         gnutls_anti_replay_t anti_replay;
 
         bool is_client;
+        bool _0rtt_enabled;
```

Comment:
These should be `const` to ensure (1) that they never change after construction, but more interestingly, (2) guaranteeing that they get initialized during construction.